### DISCUSSION TITLE
get the current viewer size after resizing with the mouse

### DISCUSSION THREAD

**Discussion by xarthurx** [libigl/libigl#1850] [Q&A] [answered]
This should be a simple question but please forgive me for asking since I cannot find it anywhere in the `Viewer` class...

I'm arranging the menu based on the left side of the viewer. It goes well when starting the viewer.
But after I resize the viewer window using the mouse, the ImGui menu is not following.

I need the new viewer size, which I guess belongs to the `glfwWindow` object? Or I was expecting some methods inside the `Viewer` class can do.

But there seems not...

So, simply put, how should I get the current viewer size?

**Comment by xarthurx** [comment, marked as answer]
OK, I found a solution
```
  glfwGetWindowSize(viewer->window, &winW, &winH);
```

If there's sth neater, please let me know.

Or if you'd like a PR for a method in the `Viewer` class, I'm also happy to contribute.

**Reply by jdumas** [reply]
It's perfectly fine to use the glfw function for that, which is why the viewer class exposes the glfw window in the first place.

**Reply by xarthurx** [reply]
Do you think we should add a method in the `Viewer` class, for users like me that are not familiar with openGL things?

**Reply by jdumas** [reply]
I'm not opposed to the idea.
